components/gmail/actions/add-label-to-email/add-label-to-email.mjs

```diff
@@ -0,0 +1,34 @@
+import common from "../../common/verify-client-id.mjs";
+
+export default {
+  ...common,
+  key: "gmail-add-label-to-email",
+  name: "Add Label to Email",
+  description: "Add a label to an email message. [See the docs](https://developers.google.com/gmail/api/reference/rest/v1/users.messages/modify)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ...common.props,
+    message: {
+      propDefinition: [
+        common.props.gmail,
+        "message",
+      ],
+    },
+    label: {
+      propDefinition: [
+        common.props.gmail,
+        "label",
+      ],
+      withLabel: true,
+    },
+  },
+  async run({ $ }) {
+    const response = await this.gmail.addLabelToEmail({
+      message: this.message,
+      label: this.label.value,
+    });
+    $.export("$summary", `Successfully added ${this.label.label} label to email`);
+    return response;
+  },
```

Comment:
Can at least the `run` method be imported directly from the other app's original action, to avoid duplicating the code?